### PR TITLE
Update ha_category to Energy

### PR DESCRIPTION
**Pull request by griddyadmin**
I noticed that this integration is currently in the Sensor category -- it really belongs in the Energy category as it is presenting the 5-minute interval pricing for Electricity in the Texas deregulated market.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/griddy.markdown [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
 title: Griddy Power
-description: Instructions on how to integrate griddy prices into Home Assistant.
+description: Instructions on how to integrate griddy real-time electricity prices into Home Assistant.
 ha_category:
-  - Sensor
+  - Energy
```
Comment (frenck) [inline source/_integrations/griddy.markdown]: I think it belongs in both. It provides sensors...

File: source/_integrations/griddy.markdown [hunk 2]
```diff
@@ -15,7 +15,7 @@ The `griddy` integration allows you to integrate your [Griddy](https://griddy.co
 
 There is currently support for the following device types within Home Assistant:
 
-- Sensor
+- Energy
```
Comment (frenck) [inline source/_integrations/griddy.markdown]: Please revert this, as it describes device platform types, not category.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/griddy.markdown (hunk 1 above)
Comment (griddyadmin) [reply source/_integrations/griddy.markdown]: Cool

File: source/_integrations/griddy.markdown (hunk 2 above)
Comment (griddyadmin) [reply source/_integrations/griddy.markdown]: Whoops...got edit happy.

**Review by frenck — APPROVED** [review]
Thanks, @griddyadmin 👍

**Pull request merged**